node_modules/oae-tenants/tests/test-tenants.js

```diff
@@ -218,6 +241,65 @@ describe('Tenants', function() {
         });
 
         /**
+         * Test that verifies a tenant can be looked up by an email domain match
+         */
+        it('verify get tenant by email domain', function(callback) {
+            var exactMatchTenantAlias = TenantsTestUtil.generateTestTenantAlias();
+            var exactMatchTenantDescription = TestsUtil.generateRandomText();
+            var exactMatchTenantHost = TenantsTestUtil.generateTestTenantHost();
+            var exactMatchTenantOpts = {'emailDomain': TenantsTestUtil.generateTestTenantHost()};
```

Comment:
Can we use `exactMatchTenantHost` as the `emailDomain` value as well? That makes it slightly easier to look at logs/debug values.